scanpipe/models.py

```diff
@@ -118,7 +118,7 @@ def task_state(self):
         return AsyncResult(str(self.task_id)).state
 
     @property
-    def execution_time(self):
+    def execution_time(self) -> int:
```

Comment:
Not sure we want to mix and match type annotations. We should probably remove this.